src/__tests__/index.test.ts

```diff
@@ -363,6 +363,13 @@ const vectors: Array<TestVector> = [
       { text: 'AGmV3oHqzfAs3VFiqmn6cecxCXVNyg6tNh', hex: '0ae542fee226c044dc19b036db7cec939777596f' },
     ],
   },
+  {
+    name: 'ARDR',
+    coinType: 1200,
```

Comment:
Can you let me know where did you get the cointype? According to SLIP0044, it's 16754

https://github.com/satoshilabs/slips/blob/master/slip-0044.md